tests/ert3/console/integration/test_cli.py

```diff
@@ -121,3 +122,129 @@ def test_cli_record_load_not_existing_file(tmpdir):
     with patch.object(sys, "argv", args):
         with pytest.raises(SystemExit):
             ert3.console.main()
+
+
+def test_cli_status_no_runs(tmpdir, capsys):
+    workspace = tmpdir / _POLY_WORKSPACE_NAME
+    shutil.copytree(_POLY_WORKSPACE, workspace)
+    workspace.chdir()
+
+    experiments = ert3.workspace.get_experiments(workspace)
+
+    args = ["ert3", "init"]
+    with patch.object(sys, "argv", args):
+        ert3.console.main()
+
+    args = ["ert3", "status"]
+    with patch.object(sys, "argv", args):
+        ert3.console.main()
+
+    captured = capsys.readouterr()
+
+    assert "have run" not in captured.out
+    assert "can be run" in captured.out
+
+    lines = [
+        line.strip()
+        for line in captured.out.splitlines()
+        if "have run" not in line and "can be run" and len(line.strip())
+    ]
+    words = (" ").join(lines).split()
+
+    for experiment in experiments:
+        assert experiment in words
+
+
+def test_cli_status_some_runs(tmpdir, capsys):
+    workspace = tmpdir / _POLY_WORKSPACE_NAME
+    shutil.copytree(_POLY_WORKSPACE, workspace)
+    workspace.chdir()
+
+    experiments = list(ert3.workspace.get_experiments(workspace))
+
+    args = ["ert3", "init"]
+    with patch.object(sys, "argv", args):
+        ert3.console.main()
+
+    done_indices = [1, 3]
+    for idx in done_indices:
+        ert3.storage.init_experiment(
+            workspace=workspace, experiment_name=experiments[idx], parameters=[]
+        )
+
+    args = ["ert3", "status"]
+    with patch.object(sys, "argv", args):
+        ert3.console.main()
+
+    captured = capsys.readouterr()
+
+    assert "have run" in captured.out
+    assert "can be run" in captured.out
+
+    lines = [
+        line.strip()
+        for line in captured.out.splitlines()
+        if "have run" not in line and "can be run" and len(line.strip())
+    ]
+
+    done = (" ").join(lines[: len(done_indices)]).split()
+    pending = (" ").join(lines[len(done_indices) :]).split()
+
+    for idx, experiment in enumerate(experiments):
+        if idx in done_indices:
+            assert experiment in done
+        else:
+            assert experiment in pending
+
+
+def test_cli_status_all_run(tmpdir, capsys):
+    workspace = tmpdir / _POLY_WORKSPACE_NAME
+    shutil.copytree(_POLY_WORKSPACE, workspace)
+    workspace.chdir()
+
+    experiments = ert3.workspace.get_experiments(workspace)
+
+    args = ["ert3", "init"]
+    with patch.object(sys, "argv", args):
+        ert3.console.main()
+
+    for experiment in experiments:
+        ert3.storage.init_experiment(
+            workspace=workspace, experiment_name=experiment, parameters=[]
+        )
+
+    args = ["ert3", "status"]
+    with patch.object(sys, "argv", args):
+        ert3.console.main()
+
+    captured = capsys.readouterr()
+
+    assert "have run" in captured.out
```

Comment:
Perhaps this entire checking could be moved into an assert function checking that some specified experiments have ran and some have not? It could then be reused by all three tests...